apps/storefront/src/utils/b3CallbackManager.ts

```diff
@@ -84,3 +90,5 @@ export default class CallbackManager {
     return success;
   }
 }
+
+export default new CallbackManager();
```

Comment:
Is this a good idea?
Seems like it'd make it harder to unit test + not sure if this behaviour will be consistent.

Reply:
This changed was required because the b2b object is created under an `useEffect` and is listening to the `b2btoken` and i was losing the callbacks if i set up them before logging into the application.

Reply:
You can see what Fani is referring to in the HeadlessController.tsx file. Basically, we create this window object in this effect that depends on the B2BToken in order to serve it in other functions, so this made the effect run when login logout happens and reset the class instance losing all your callbacks. 

By using exporting this singleton, we maintain the references across all renders.

Reply:
@deov31 - can we solve that issue another way.

i.e create the singleton outside of useEffect & the component.